### PR TITLE
Team 1 introcutsceens

### PR DESCRIPTION
**Pull request by yashwanthg22**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes / Closes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by IanP96 — CHANGES_REQUESTED** [review]
please see comments and fix the button issue

File: source/core/src/main/com/csse3200/game/components/screendisplay/IntroCutsceneDisplay.java [hunk 1]
```diff
@@ -1,117 +1,160 @@
...
-import com.csse3200.game.screens.IntroCutsceneScreen;
 import com.csse3200.game.ui.UIComponent;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import com.badlogic.gdx.Input;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.InputListener;
 
-/**
- * Creates the ui elements and functionality for the intro cutscene's ui.
```
[the remark this answers is not in the document]
Comment (yashwanthg22) [reply source/core/src/main/com/csse3200/game/components/screendisplay/IntroCutsceneDisplay.java:18]: I added the comments back.

File: source/core/src/main/com/csse3200/game/components/screendisplay/IntroCutsceneDisplay.java [hunk 2]
```diff
@@ -1,117 +1,160 @@
...
     private final GdxGame game;
     private Table table;
+    private int currentPage = 0;
+
+    private Texture[] cutsceneTextures;
+    private Image backgroundImage;
 
-    /**
-     * Make the component.
-     * @param game the overarching game, needed so that buttons can trigger navigation through
```
[the remark this answers is not in the document]
Comment (yashwanthg22) [reply source/core/src/main/com/csse3200/game/components/screendisplay/IntroCutsceneDisplay.java]: i added the comments back.

File: source/core/src/main/com/csse3200/game/components/screendisplay/IntroCutsceneDisplay.java [hunk 3]
```diff
@@ -1,117 +1,160 @@
...
+            }
+        });
+
+        updateCutscene(); // Start with the first page
+    }
+
+    private void loadTextures() {
+        cutsceneTextures = new Texture[4]; // Adjusted size based on the number of cutscenes
+        cutsceneTextures[0] = new Texture(Gdx.files.internal("images/screen/cutscene1.jpg"));
+        cutsceneTextures[1] = new Texture(Gdx.files.internal("images/screen/cutscene2.jpg"));
```
[the remark this answers is not in the document]
Comment (yashwanthg22) [reply source/core/src/main/com/csse3200/game/components/screendisplay/IntroCutsceneDisplay.java]: i have added a loop for jpg paths.

**Pull request merged**